setup.py

```diff
@@ -119,7 +119,8 @@ def run(self):
              'scripts/test_some_write_keys_others_read_them',
              'scripts/test_users_write_and_read_own_keys',
              'scripts/validator-info',
-             'scripts/init_bls_keys'],
+             'scripts/init_bls_keys',
+             'scripts/init_bls_for_stewards'],
```

Comment:
This name is confusing.
As far as I understand this script creates bls keys and sends transaction to publish them, but it's name tells that it adds bls keys for steward (as if steward can have bls keys)

Reply:
Renamed to `enable_bls`